packages/hooks/src/useHasSetCurrency.ts

```diff
@@ -5,9 +5,10 @@ const useHasSetCurrency = () => {
     const { client } = useStore();
     const { account_list } = client;
     const has_active_real_account = useHasActiveRealAccount();
+
     const has_real_account = account_list
         .filter(account => !account.is_virtual)
-        .some(account => account.title !== 'Real');
+        .some(account => account.title !== 'Real' && account.title !== 'Investment');
```

Comment:
@thisyahlen-deriv Would be nice to add more test cases to cover this title check too 👀

Reply:
Done

Reply:
Awesome, @mohsen-deriv should give you a raise 🚀😁